src/Button/index.scss

```
@@ -97,3 +97,13 @@
 .nds-plain-button {
   @extend %btn-kind-plain;
 }
+
+.nds-button--spread {
+  justify-content: stretch !important;
+
+  &.nds-button--xs .nds-button-content,
+  &.nds-button--s .nds-button-content,
+  &.nds-button--m .nds-button-content {
+    width: 100% !important;
+  }
```

I think what we want to happen with `isSpread` is for the root element (the button or anchor) takes up 100% of parent width
```suggestion
    display: flex; /* instead of inline flex , makes it behave like display: block */
  }
```